packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -192,7 +193,7 @@ export default defineComponent({
     ...useFormProps,
 
     modelValue: {
-      type: [String, Number, Object, Array] as PropType<string | number | Record<string, any> | any[]>,
+      type: [String, Number, Object] as PropType<SelectableOption>,
```

Comment:
Hey @LighthouseKeeperYN 

Could you tell why `Array` type was removed from `VaSelect` `modelValue`?

Because it has Array as its value most of the time.